modules/jdbc-test/src/test/java/org/testcontainers/junit/SimplePostgreSQLTest.java

```diff
@@ -17,23 +17,42 @@
  */
 public class SimplePostgreSQLTest {
 
-    @Rule
-    public PostgreSQLContainer postgres = new PostgreSQLContainer();
-
     @Test
     public void testSimple() throws SQLException {
+        try (PostgreSQLContainer postgres = new PostgreSQLContainer<>()) {
+            postgres.start();
+
+            ResultSet resultSet = performQuery(postgres, "SELECT 1");
+
+            int resultSetInt = resultSet.getInt(1);
+            assertEquals("A basic SELECT query succeeds", 1, resultSetInt);
+        }
+    }
+
+    @Test
+    public void testExplicitInitScript() throws SQLException {
+        try (PostgreSQLContainer postgres = new PostgreSQLContainer<>()
+                .withInitScript("somepath/init_postgresql.sql")) {
+            postgres.start();
+
+            ResultSet resultSet = performQuery(postgres, "SELECT foo FROM bar");
+
+            String firstColumnValue = resultSet.getString(1);
+            assertEquals("Value from init script should equal real value", "hello world", firstColumnValue);
+        }
+    }
+
+    private ResultSet performQuery(JdbcDatabaseContainer container, String sql) throws SQLException {
```

Comment:
I have a strong desire to extract this code to separate TestUtils and use it here and in SimpleMySQLTest.

Also why no connections and result sets here and in SimpleMySQLTest are closes? Of course it's all tests, but still.